python/pyarrow/tests/test_substrait.py

```diff
@@ -66,11 +73,9 @@ def test_run_serialized_query(tmpdir):
     }
     """
     # TODO: replace with ipc when the support is finalized in C++
```

Comment:
Is this comment still valid?  It looks like this is IPC data.

Reply:
This needs to be removed.